# polii.gene.call

Rscript for calculating average PolII occupancy and FDR for RNA Pol II DamID datasets.  Based on original algorithms developed by Tony Southall as published in Southall et al. (2013). Dev Cell, 26(1), 101â€“12. doi:10.1016/j.devcel.2013.05.020.  (Modifications from the original method are described in detail in the sourcecode.)

The script processes datafiles in gatc.bed or gatc.gff format, such as those generated by [damidseq_pipeline](https://owenjm.github.io/damidseq_pipeline).

## Requirements

  1. R
  3. A GFF-formatted list of genes.  A file for release 6 of the Drosophila genome is provided in the archive; most GFF annotation files should also work.  Place this file in an accessible directory and use the --genes.file commandline switch to access it:

		polii.gene.call.r --genes.file=/path/to/my-genes-anotation.gff

## Installation

To install, copy the polii.gene.call.r executable into your path.

## Usage

Run polii.gene.call.r as follows:

    polii.gene.call.r [options] [list of gatc.bed and/or gatc.gff files to process]

Each file will be processed separately, with the output being two files:

  1. [name].genes.details.csv
  	
	A .csv table of all genes, together with average occupancy and FDR
  2. [name].genes
  	
	A plain text list of all genes below the FDR threshold (default is 0.01; change with --FDR= commandline switch).  These genes may be considered to represent the significantly transcribed genes within a genome.

For a list of all possible commandline options, use

    polii.gene.call.r --help

## Downstream data processing

Two different transcriptomes generated through this method may be compared using the [polii.correlation.plot](https://github.com/owenjm/damid_misc/blob/master/polii.correlation.plot) available from the [damid_misc](https://github.com/owenjm/damid_misc) repository.

		Rscript polii.correlation.plot [file1.genes.details.csv] [file2.genes.details.csv]

The output is both a graphical plot of differentially expressed genes, and a table listing the difference in mean log2(occupancy).

## Differences between RNA pol II DamID and RNAseq

Although both are methods for transcriptional profiling, please be aware that there may be differences between these two methods.  In particular, transcript abundancy as assessed through RNAseq will depend on transcript stability, whereas RNA pol II occupancy may provide a better indication of transcription levels.
